tests/sentry/api/endpoints/test_organization_missing_org_members.py

```diff
@@ -56,6 +56,12 @@ def setUp(self):
         self.integration = self.create_integration(
             organization=self.organization, provider="github", name="Github", external_id="github:1"
         )
+        self.integration2 = self.create_integration(
```

Comment:
the error shows up when you have more than 1 integration. adding one here fails tests in this file without the changes made here